services/nvidia-gpu-operator/24.3.2/helmrelease/extra-images.txt

```diff
@@ -4,4 +4,4 @@ ghcr.io/mesosphere/dkp-container-images/nvcr.io/nvidia/cloud-native/gpu-operator
 nvcr.io/nvidia/cloud-native/dcgm:{{ .Values.dcgm.version }}
 nvcr.io/nvidia/k8s/dcgm-exporter:{{ .Values.dcgmExporter.version }}
 ghcr.io/mesosphere/dkp-container-images/nvcr.io/nvidia/k8s-device-plugin:{{ .Values.devicePlugin.version }}
-nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda10.2
+nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda12.5
```

Comment:
This needs to be `nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda12.5.0`

```suggestion
nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda12.5.0
```

The `nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda12.5` doesn't exist in the registry.

Reply:
right, the ref throws error
https://api.github.com/repos/NVIDIA/cuda-samples/contents/?ref=v12.5.0